src/sentry/tasks/merge.py

```diff
@@ -163,17 +189,17 @@ def _rehash_group_events(group, limit=100):
 
 
 def merge_objects(models, group, new_group, limit=1000,
-                  logger=None):
+                  logger=None, first_pass=True, transaction_id=None):
     from sentry.models import GroupTagKey, GroupTagValue
 
     has_more = False
+    if first_pass:
+        logger.info('objects.merge.executed', extra={
```

Comment:
The bad part about this is it now literally says nothing. It doesnt' say what objects, it just repeats the fact that these two groups are merging. So I'd vote to just remove this, or move it outside into `merge_group`.